api/backend.proto

```diff
@@ -75,7 +75,7 @@ message FetchMatchesRequest {
 }
 
 message FetchMatchesResponse {
-  // Result Match for the requested MatchProfile.
+  // Result Match for the requested MatchProfile. Note that open-match will validate the proposals returned from MMF and skip the ones with empty tickets.
```

Comment:
nit: Split comment to next line. also can simply state - a valid match should contain atleast one ticket.

Also, We should add that comment actually to the Match proto too.